net/net.go

```diff
@@ -386,7 +386,6 @@ func (n *net) PullThread(ctx context.Context, id thread.ID, opts ...core.ThreadO
 }
 
 func (n *net) pullThread(ctx context.Context, id thread.ID) error {
-	log.Debugf("pulling thread %s...", id)
```

Comment:
This creates too much output, to the point where you can't easily make sense of the logs.